test/src/scenes/test.tsx

```diff
@@ -277,8 +300,9 @@ export default makeScene2D(function* (view) {
 
   yield* waitFor(2);
 
-  yield txt().text('Color Themes', 1);
-  yield* c().style(materialHighlightStyle, 1);
+  // TODO: Reimplement via CodeHighlighter
```

Comment:
Can we try swapping the highlighter here, even if it's just another Lezer highlighter with the other theme? It sounded like that was supported based on our discussion in Discord.

This can be another PR / issue if need be.

Reply:
Done